recipes/caiman/meta.yaml

```
@@ -0,0 +1,79 @@
+{% set name = "CaImAn" %}
+{% set version = "1.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/flatironinstitute/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ab272a4cd8ed4200fd844c7a9fa5770257313a8f6eb658ce57a82577f0ca3e75
+
+build:
+  number: 0
+  skip: True  # [not py==36]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python
+    - pip
+    - numpy
+    - cython
+    - ripgrep
+  run:
+    - python=3.6
+    - tensorflow
+    - bokeh
+    - cython
+    - future
+    - h5py
+    - ipykernel=4.10
+    - ipython
+    - ipyparallel
+    - jupyter
+    - matplotlib
+    - mypy
+    - nose
+    - numpy
+    - opencv
+    - peakutils
+    - pims
+    - scipy
+    - scikit-image
+    - scikit-learn
+    - spyder
+    - tifffile
+    - tk=8.6.8
```

Space and does this need to be so strict?
```suggestion
    - tk =8.6.8
```

---

@chrisburr thanks. I pinned it to less than or equal to 8.6.8, because 8.6.9 (latest on conda-forge) is buggy for a particular feature we are using. It didn't seem like there was a way to explicitly exclude a single version of a package in a conda dependencies specification. Do you know of one, by any chance?

---

@jjhelmus helped me out!